midterm-2023-11-07/code/4660-darko-ivanovski/script.ts

```diff
@@ -0,0 +1,154 @@
+document.addEventListener("DOMContentLoaded", listOfMovies);
+
+interface Cast
+{
+    actor: string;
+    character: string;
+}
+
+interface Oscars
+{
+    bestPicture: string;
+    bestDirector: string;
+    bestActor: string;
+    bestAdaptedScreenplay: string; 
+}
+
+interface Movie
```

Comment:
The oscars field is really not an array, it's an object. Otherwise, the types are correct. +8